### PR TITLE
Fixes weird unicode rendering in command announcements

### PR DESCRIPTION
**Pull request by BeloneX**
## About The Pull Request

e.g ' gets replaced with & # 39; and all that

image
Proof that it works.

ports: https://github.com/BeeStation/BeeStation-Hornet/pull/3567
## Why It's Good For The Game

it's a fix
## Changelog 
:cl: jupyterkat BeloneX
fix: fixed weird announcement text renderings
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
You should wait until PRs are merged before porting to save yourself the hassle of changing stuff if they do

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**
